### PR TITLE
Add a CONTRIBUTING.md page

### PR DESCRIPTION
**Pull request by CdrMarks**
Based on my work on #111, I put this guide together.

### CODE REVIEW

**Review by jabxjab — APPROVED** [review]

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#Testing
```
Comment (jspellman814) [inline docs/CONTRIBUTING.md:1]:
```suggestion
# Testing
```

**Review by jspellman814 — APPROVED** [review]
@CdrMarks one small change needed, otherwise LGTM.

**Comment by namespacebrian** [comment]
Did I do this wrong?  it's telling me the branches "- *" is invalid: https://github.com/pantheon-systems/wp_launch_check/actions/runs/2073199374/workflow

**Review by namespacebrian — APPROVED** [review]
I've run through this process and although I'm not quite sure how to test my tagged deploy (I think I'd need a valid WPScan token?), and despite the weird error about the `- *` branches specification, I'm sufficiently confident that the process worked as intended.

**Comment by CdrMarks** [comment]
I figured out the syntax for every branch.  It should have been 
`- '**'`

**Review by jspellman814 — APPROVED** [review]

**Pull request merged**